src/filters/string.rs

```diff
@@ -116,4 +135,34 @@ mod tests {
         assert!(result.is_ok());
         assert_eq!(result.unwrap(), to_value(&4));
     }
+
+    #[test]
+    fn test_replace() {
```

Comment:
you only need that test since we are using the built-in `replace`, you can remove the 2 other tests